plotting/utils.py

```
@@ -77,7 +77,7 @@ def mathtext(text):
 
 
 # Plots point(s) on a map (called when "Show Location" is true)
-def _map_plot(points, path=True, quiver=True):
+def _map_plot(points, path=True, quiver=True, colors=list()):
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)